tests/snapshot-gating.fmf

```diff
@@ -36,6 +36,9 @@ prepare:
       # already in the system.
       - dnf -y install --best llvm-libs
 
+      # Use a newer llvm-test-suite version from copr.
+      - dnf copr enable -y @fedora-llvm-team/llvm-test-suite $COPR_CHROOT
```

Comment:
@jchecahi I think we have another reason for lowering the testing-farm repo instead of raising our own: We now have two repos to add ;)

@nikic This looks good to me. But I wonder what will happen if we for example add a new chroot to the snapshot builds, we would need to add it here as well, right? For example, `centos-stream-10-...` is missing from the list in https://copr.fedorainfracloud.org/coprs/g/fedora-llvm-team/llvm-test-suite/monitor/.

Reply:
Yes, we'd have to add the new chroots here, similar to how we do in llvm-compat-packages. Though we don't need to do anything for new fedora versions, as those get forked automatically.

I can go ahead and add the cs10 chroots now.

Reply:
The cs10 chroots are now present: https://copr.fedorainfracloud.org/coprs/g/fedora-llvm-team/llvm-test-suite/monitor/

Reply:
> The cs10 chroots are now present: https://copr.fedorainfracloud.org/coprs/g/fedora-llvm-team/llvm-test-suite/monitor/

@nikic awesome. Thank you!